CRM/Upgrade/Incremental/php/FiveFortyOne.php

```diff
@@ -85,6 +85,42 @@ public function upgrade_5_41_alpha1($rev) {
     );
   }
 
+  /**
+   * Upgrade function.
+   *
+   * @param string $rev
+   */
+  public function upgrade_5_41_beta1($rev) {
+    $this->addTask('Ensure non-English installs have a default financial type for the membership price set.',
+      'ensureDefaultMembershipFinancialType');
+  }
+
+  public static function ensureDefaultMembershipFinancialType(): bool {
+    $membershipPriceSet = CRM_Core_DAO::executeQuery(
+      'SELECT id, financial_type_id FROM civicrm_price_set WHERE name = "default_membership_type_amount"'
+    );
+    $membershipPriceSet->fetch();
+    if (!is_numeric($membershipPriceSet->financial_type_id)) {
+      $membershipFinancialTypeID = CRM_Core_DAO::singleValueQuery('
+        SELECT id FROM civicrm_financial_type
+        WHERE name = "Member Dues"
+        OR name = %1', [1 => [ts('Member Dues'), 'String']]
+      );
+      if (!$membershipFinancialTypeID) {
+        // This should be unreachable - but something is better than nothing
```

Comment:
Thanks @eileenmcnaughton for doing this. It all looks good.

Just noting this is reachable if you install in non-english and then change the financial type "label" (name), but that doesn't change anything for what the PR is able to do.